core/objects.go

```diff
@@ -20,6 +20,25 @@ import (
 	"github.com/square/go-jose"
 )
 
+// MaxCNLength is the maximum length allowed for the common name as specified in RFC 5280
```

Comment:
Can you put these values in csrlib instead? We should be very cautious about adding new things to core (and I want to take some things out of core too).